src/main/java/com/github/onsdigital/babbage/configuration/Configuration.java

```diff
@@ -138,6 +143,29 @@ public static Integer getElasticSearchPort() {
         public static String getElasticSearchCluster() {
             return elasticSearchCluster;
         }
+
+        /**
+         * Method to load the list of retired product pages to be hidden
+         * @return List of url strings containing the black listed urls
+         */
```

Comment:
The logic of this method is fine - however its an unnecessary overhead to do this every time this method is called. I would implement a thread safe singleton pattern here - that way its loaded once and the method can return the populated list in subsequent calls.